package.json

```diff
@@ -44,19 +43,12 @@
     "@types/ws": "^7.2.4",
     "@typescript-eslint/eslint-plugin": "^4.3.0",
     "@typescript-eslint/parser": "^4.3.0",
-    "commitizen": "^4.2.1",
-    "cz-conventional-changelog": "^3.3.0",
     "eslint": "^7.10.0",
     "eslint-config-prettier": "^6.12.0",
     "eslint-plugin-prettier": "^3.1.4",
     "nodemon": "^2.0.4",
     "prettier": "^2.1.2",
     "ts-node-dev": "^1.0.0-pre.62",
     "typescript": "3.5.3"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
```

Comment:
How come `commitizen` and `cz-conventional-changelog` were removed? I think we still need them

Reply:
I think I screwed up the file due to the dependency changes